static/js/ducks/instruments.js

```diff
@@ -15,11 +24,17 @@ const reducer = (state = { instrumentList: [] }, action) => {
         instrumentList: instruments,
       };
     }
+    case FETCH_INSTRUMENT_FORMS_OK: {
+      const instrumentFormParams = action.data;
+      return {
+        ...state,
+        instrumentFormParams,
+      };
+    }
     default:
       return state;
   }
 };
 
-store.injectReducer("instruments", reducer);
-
 export default fetchInstruments;
```

Comment:
Now that there's more than one export, don't specify a default export
```suggestion
```

Reply:
got it. thanks

Reply:
fxied